src/05-derivatives/02-common-data-types.md

```diff
@@ -0,0 +1,58 @@
+# Common data types
+
+## Preprocessed or cleaned data
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        <datatype>/
+            <source_keywords>[_space-<space>][_desc-<label>]_<suffix>.<ext>
+```
+
+Preprocessing in this context means transformations of data that do not change
+the data type of the input (as expressed by its BIDS `suffix`).
+For instance, a change in the number of dimensions is likely to disrupt the propagation
+of the input's `suffix` and generally, the outcomes of such transformation
+cannot be considered _preprocessed_ or _cleaned data_.
+Examples:
```

Comment:
Since the sentence preceding "Examples" is about what is *not* considered preprocessed, it is ambiguous whether the following are examples for either preprocessed data, or examples that do not qualify as preprocessed data (i.e., the opposite). 

I recommend writing `Examples of preprocessed data:`